bilingual-module/README

```diff
@@ -26,49 +27,60 @@ if_hfst}}
 
 If this does not make any sense, we recommend you look at: www.apertium.org.
 
+
 Compiling
 -------------------------------------------------------------------------------
 
 Given the requirements being installed, you should be able to just run:
 
-    $ ./configure
-    $ make
-    # make install
+```bash
+$ ./configure
+$ make
+$ make install
+```
+
+You can use `./autogen.sh` instead of `./configure` in case you're compiling from
+source (GitHub).
+If you installed any prerequisite language packages using a `--prefix`
+with `./configure`, make sure to use the same `--prefix` when running `./configure` here.
 
-You can use `./autogen.sh` instead of `./configure` you're compiling from
```

Comment:
the "in case" is missing.

Reply:
What do you mean? I added it in the commit, it wasn't there previously.